app/services/platforms/facebook.ts

```diff
@@ -36,6 +43,13 @@ export class FacebookService extends StatefulService<IFacebookServiceState>
   @Inject() settingsService: SettingsService;
   @Inject() userService: UserService;
 
+  capabilities = new Set([
+    'chat',
+    'user-info',
+    'viewer-count',
+    'stream-schedule',
+  ] as TPlatformCapability[]);
```

Comment:
Can we avoid the typecast here?  TypeScript supports generic sets: `new Set<TPlatformCapability>(...)`